client/src/node/extension.ts

```diff
@@ -80,13 +82,15 @@ export function activate(context: ExtensionContext): void {
   context.subscriptions.push(
     commands.registerCommand("SAS.run", run),
     commands.registerCommand("SAS.runSelected", runSelected),
-    commands.registerCommand("SAS.close", (silent) =>
-      closeSession(silent === true ? undefined : "The SAS session has closed.")
-    ),
+    commands.registerCommand("SAS.close", (silent) => {
+      closeSession(silent === true ? undefined : "The SAS session has closed.");
+      commands.executeCommand("setContext", "SAS.authorized", false);
```

Comment:
I'm thinking we don't need this line here. AuthProvider set it to false when sign out that looks sufficient to me.

Reply:
Ugh...missed that one. Should be removed now :)